util/src/main/java/org/datacommons/util/StatChecker.java

```diff
@@ -361,4 +385,47 @@ protected static void checkDates(
       prev = dt;
     }
   }
+
+  // Return false if we have already seen the same StatVarObservation node but with a different
+  // value.
+  private boolean checkSvObsValueInconsistency(McfGraph.PropertyValues node) {
+    String placeDcid = McfUtil.getPropVal(node, Vocabulary.OBSERVATION_ABOUT);
+    String statVarDcid = McfUtil.getPropVal(node, Vocabulary.VARIABLE_MEASURED);
+    String mmethod = McfUtil.getPropVal(node, Vocabulary.MEASUREMENT_METHOD);
+    String obsPeriod = McfUtil.getPropVal(node, Vocabulary.OBSERVATION_PERIOD);
+    String sFactor = McfUtil.getPropVal(node, Vocabulary.SCALING_FACTOR);
+    String unit = McfUtil.getPropVal(node, Vocabulary.UNIT);
+    String obsDate = McfUtil.getPropVal(node, Vocabulary.OBSERVATION_DATE);
+    Hasher hasher = Hashing.farmHashFingerprint64().newHasher();
+    for (String prop :
+        List.of(placeDcid, statVarDcid, mmethod, obsPeriod, sFactor, unit, obsDate)) {
+      if (prop.isEmpty()) {
+        hasher.putString(EMPTY_PROP_STRING, StandardCharsets.UTF_8);
+      } else {
+        hasher.putString(prop, StandardCharsets.UTF_8).putInt(prop.length());
+      }
+    }
```

Comment:
Would it be more readable to restructure as:

```
private static SVOBS_VALUE_KEY_PROPS = List.of(Vocabulary.OBSERVATION_ABOUT, ...);
  
  Hasher hasher = Hashing.farmHashFingerprint64().newHasher();
  for (var p : SVOBS_VALUE_KEY_PROPS) {
    var v = McfUtil.getPropVal(node, p);
    hasher.putString(v.isEmpty() ? EMPTY_PROP_STRING : v, UTF_8);
  }
  ...
```

Reply:
done